ehrapy/preprocessing/_normalization.py

```diff
@@ -396,3 +403,35 @@ def _record_norm(adata: AnnData, vars_list: list[str], method: str) -> None:
     adata.uns["normalization"] = norm_record
 
     return None
+
+
+def offset_negative_values(adata: AnnData, layer: str = None, copy: bool = False) -> AnnData:
```

Comment:
Add `var_names` parameter? So users can pass column names, with negative values; if this is None -> use function as it is now.

Reply:
Was my o.g. plan, but not sure whether should be possible in the first place. Either all or none?

Reply:
Yeah valid. Was just thinking whether this could be a "speed improvement" when there is for example only one column, that should be corrected (the only one with negative values), as it's the case in the pediatric ICU dataset.